src/cs-utils/FrameTimings.cpp

```diff
@@ -156,8 +157,9 @@ void TimerQueryPool::start(std::string const& name, FrameTimings::QueryMode mode
     if (t) {
       range.mGPUStart = *t;
     } else {
-      std::cerr << "WARN: TimerQueryPool::Start, No more Timestamps available (mMaxSize="
-                << mMaxSize << ", mIndex=" << mIndex << "\n";
+      spdlog::error(
```

Comment:
The original was a warning. Should this too?

```suggestion
      spdlog::warn(
```